modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -74,125 +86,165 @@ get.parameter.samples <- function(settings, ensemble.size = 1,
+      q_samples <- PEcAn.emulator::lhc(
+        t(matrix(0:1, ncol = length(priors), nrow = 2)),
+        samples.num
+      )
     } else if (ens.sample.method == "uniform") {
-      q_samples <- matrix(stats::runif(samples.num * length(priors)),
-                               samples.num, 
-                               length(priors))
+      q_samples <- matrix(
+        stats::runif(samples.num * length(priors)),
+        samples.num,
+        length(priors)
+      )
     } else {
-      PEcAn.logger::logger.info("Method ", ens.sample.method, " has not been implemented yet, using uniform random sampling")
+      PEcAn.logger::logger.info(
+        "Method ", ens.sample.method,
+        " has not been implemented yet, using uniform random sampling"
+      )
       # uniform random
-      q_samples <- matrix(stats::runif(samples.num * length(priors)),
-                          samples.num, 
-                          length(priors))
+      q_samples <- matrix(
+        stats::runif(samples.num * length(priors)),
+        samples.num,
+        length(priors)
+      )
     }
     for (prior in priors) {
       if (prior %in% param.names[[i]]) {
         samples <- distns$trait.mcmc[[prior]] %>%
-          purrr::map(~ .x[,'beta.o']) %>%
+          purrr::map(~ .x[, "beta.o"]) %>%
           unlist() %>%
           as.matrix()
       } else {
-        samples <- PEcAn.priors::get.sample(distns$prior.distns[prior, ], samples.num, q_samples[ , priors==prior])
+        samples <- PEcAn.priors::get.sample(
+          distns$prior.distns[prior, ],
+          samples.num,
+          q_samples[, priors == prior]
+        )
       }
       trait.samples[[pft.name]][[prior]] <- samples
     }
-  }  ### End for loop
-  
+  } ### End for loop
+
   # if samples are independent, set param.names to NULL
-  # this is important for downstream, when param.names is not NULL MCMC will be sampled accordingly
-  if(independent){
+  # this is important for downstream, when param.names is not NULL
+  # MCMC will be sampled accordingly
+  if (independent) {
     param.names <- NULL
   }
-  
+
   if ("sensitivity.analysis" %in% names(settings)) {
-    
-    ### Get info on the quantiles to be run in the sensitivity analysis (if requested)
-    quantiles <- PEcAn.utils::get.quantiles(settings$sensitivity.analysis$quantiles)
+    ### Get info on the quantiles to be run in the sensitivity analysis
+    ### (if requested)
+    quantiles <- PEcAn.utils::get.quantiles(
+      settings$sensitivity.analysis$quantiles
+    )
     ### Get info on the years to run the sensitivity analysis (if requested)
-    sa.years <- data.frame(sa.start = settings$sensitivity.analysis$start.year, 
-                           sa.end = settings$sensitivity.analysis$end.year)
-    
-    PEcAn.logger::logger.info("\n Selected Quantiles: ", PEcAn.utils::vecpaste(round(quantiles, 3)))
-    
+    sa.years <- data.frame(
+      sa.start = settings$sensitivity.analysis$start.year,
```

Comment:
I'm just going to stop leaving individual comments --- stylizer is really hitting my pet peeves on this one.  Having the code indent to left of the ( makes the code harder to read. Having it indent left of the <- *REALLY* bugs me (things on the LHS needs to say on the LHS, things on the RHS needs to stay on the RHS)